### DISCUSSION TITLE
Is there a way to publish a robots.txt automatically when generating the site?

### DISCUSSION THREAD

**Discussion by jenspfahl-tgtg** [testingisdocumenting/znai#1115] [Q&A]
I want to publish a robots.txt file when generating the site (ideally also in preview mode). Just placing the file next to the meta.json doesn't work. Is there a way to tell Znai to do this?

**Comment by MykolaGolubyev** [comment]
Oh good question. Sounds like we need to add it. Any chance you can create an issue?

**Reply by jenspfahl-tgtg** [reply]
Sure: #1117

**Reply by MykolaGolubyev** [reply]
did you have a chance to try it?
